fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/RecordTypeTable.java

```diff
@@ -250,6 +250,8 @@ public static Message toDynamicMessage(RelationalStruct struct, Descriptors.Desc
                             }
                         }
                         break;
+                    case NULL:
+                        break;
```

Comment:
so a `null`-typed field is reflected by non-existing field in the respective dynamic message.